recipes/structureharvester/meta.yaml

```diff
@@ -0,0 +1,30 @@
+{% set name = "structureHarvester" %}
+{% set version = "0.6.94" %}
+{% set commit = "e15952601b946267ce566df6ff0dd56df5c8b8f6" %}
+
+package:
+  name: {{ name|lower }}
```

Comment:
Please put the name directly in here. Our goal is to only use jinja for stuff that occurs more than once and changes upon new releases.

Reply:
Done